registry/handlers/referrers.go

```diff
@@ -195,6 +225,20 @@ func readlink(ctx context.Context, path string, stDriver driver.StorageDriver) (
 	return digest.Parse(string(content))
 }
 
+func generateLinkHeader(repoName, subjectDigest, artifactType string, lastDigests []string, nPage int, p int) string {
+	url := fmt.Sprintf("/v2/%s/referrers/%s?p=%d",
+		repoName,
+		subjectDigest,
+		p)
+	if artifactType != "" {
+		url = fmt.Sprintf("%s&artifactType=%s", url, artifactType)
+	}
```

Comment:
You may consider to use `url.Value` since `artifactType` may contain special characters like `/`.

Reply:
Used `url.Values` here. Thanks for this good advice.